config/symbols.us.txt

```diff
@@ -1027,25 +1027,8 @@ g_Status_relics_27 = 0x8009797F;
 g_Status_relics_28 = 0x80097980;
 g_Status_relics_29 = 0x80097981;
 g_Status_spells = 0x80097982;
-D_8009799A = 0x8009799A;
```

Comment:
Should we have `g_Status_statsFamiliars_0_level` etc to improve the asm?

Reply:
Probably not necessary at least for now - I don't think many places in the game will really care about the level of familiars? Maybe we can revisit that later but for now it seems like that would be a lot of symbols for not much benefit. Open to other suggestions though.